uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp/tasks.js

```diff
@@ -11,18 +11,20 @@ module.exports = (function () {
 
       casper.then(function () {
         casper.click('.pane-views-tasks-block a.show-complete-tasks');
-        casper.waitUntilVisible('.modal-civihr-custom__footer');
+        casper.waitWhileVisible('.loading-spinner');
+        casper.waitUntilVisible('.view-Tasks');
       });
     },
+
     /**
      * Opens Create New Task modal
      */
     openCreateNewTaskModal: function () {
       var casper = this.casper;
 
       casper.then(function () {
-        casper.clickLabel('Create new task', 'a');
-        casper.waitUntilVisible('.modal-civihr-custom__footer');
+        casper.click('[href="/civi_tasks/nojs/create"]');
```

Comment:
Sorry, but this has the same risks as selection via label. Can we use a class/id anything/custom attribute that will not probably change in the future?

Reply:
Fixed by adding new class.